WordPress/gutenberg
# Migrate draggable block tests to Playwright

## What?
I started working on this migration in hopes of fixing #43737. Unfortunately can't get dragging work correctly.

I also tried using [`page.dragAndDrop`](https://github.com/microsoft/playwright/pull/6910) and logging the following message for failure - `subtree intercepts pointer events`.

Sharing current code, maybe a fresh perspective can help me unblock.

Cc @kevin940726, @ciampo, @talldan, @WunderBart 

## Why?


## How?


## Testing Instructions
Running the test:

```
npm run test:e2e:playwright -- test/e2e/specs/editor/various/draggable-blocks.spec.js
```

Check the trace:

```
npx playwright show-trace artifacts/test-results/editor-various-draggable-blocks-Draggable-block-can-drag-and-drop-to-the-top-of-a-block-list-chromium/trace.zip
```

## Screenshots or screencast
